packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab-header.jsx

```diff
@@ -38,6 +40,12 @@ const VerticalTabHeader = ({ children, className, is_routed, item, onChange, sel
             >
                 <HeaderIcon icon={item.icon} is_active={is_active} />
                 <Header text={label} />
+                {!!count &&
+                    <Counter
+                        count={count}
+                        className='vertical-tab__header__counter'
```

Comment:
```suggestion
                        className='vertical-tab__header-counter'
```

Reply:
This pattern for naming classes is used before in the styling of that component for `__icon` and `__link` before.

Reply:
IMO both should change then, our styleguide only allows one level of `__` nesting